src/player.js

```diff
@@ -1866,6 +1848,23 @@ export default class Player extends FakeEventTarget {
     this._addTextTrackOffOption();
     this._maybeSetTracksLabels();
     this._setDefaultTracks();
+    this._maybeAdjustTextTracksIndexes();
```

Comment:
Isn't it better enumerating before setting default?

Reply:
fixed